tests/baselines/reference/tsxStatelessFunctionComponentsWithTypeArguments2.errors.txt

```diff
@@ -8,13 +8,15 @@ tests/cases/conformance/jsx/file.tsx(20,19): error TS2322: Type '{ func: (a: num
   Type '{ func: (a: number, b: string) => void; }' is not assignable to type '{ func: (arg: number) => void; }'.
     Types of property 'func' are incompatible.
       Type '(a: number, b: string) => void' is not assignable to type '(arg: number) => void'.
-tests/cases/conformance/jsx/file.tsx(31,9): error TS2605: JSX element type 'Element' is not a constructor function for JSX elements.
-  Property 'render' is missing in type 'Element'.

```

Comment:
@yuit or @RyanCavanaugh This error is gone in the update basedline. But it looks bogus. Is it?